Add cuQuantum Python v0.1.0.0

leofang opened the pull request:
~~Blocked by https://github.com/conda-forge/cuquantum-feedstock/pull/1.~~

<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

leofang committed:
enable building with public release

MNT: Re-rendered with conda-build 3.21.6, conda-smithy 3.16.0, and conda-forge-pinning 2021.12.18.06.45.12

leofang committed:
fix for the published package structure

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

leofang committed:
temporarily disable aarch64 builds

MNT: Re-rendered with conda-build 3.21.6, conda-smithy 3.16.0, and conda-forge-pinning 2021.12.20.19.09.43

leofang commented:
@conda-forge-admin, please restart ci
